lambda_functions/review_service/create_review_answer.py

```diff
@@ -51,6 +52,11 @@ def create_review_answer(event, context, is_test=False, session=None):
         review_answer_handler.create_review_answer(
             review_answer, is_test, session)
 
+        review = Review()
+        review = review_answer.review
+        review.last_question_id = review_answer.review_question_id
+        update_object(review, is_test, session)
```

Comment:
Do you actually have to initialize a new object?
Can't you just assign `review_answer.review.last_question_id = review_answer.review_question_id`?